VOKMockUrlProtocol.m

```diff
@@ -11,6 +11,7 @@
 
 #import <ILGHttpConstants/HTTPStatusCodes.h>
 #import <ILGHttpConstants/HTTPMethods.h>
+#import <ILGHttpConstants/HttpHeaderFields.h>
```

Comment:
Most OS X filesystems are set up case-insensitive, so it'd probably be okay, except on that rare case-sensitive filesystem.